.github/CODEOWNERS

```
@@ -0,0 +1,2 @@
+yarn.lock   * @VKCOM/vk-sec
+.yarnrc     * @VKCOM/vk-sec
```

Кажется, * тут лишние. [Вот тут про синтаксис больше](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#example-of-a-codeowners-file)

```suggestion
yarn.lock   @VKCOM/vk-sec
.yarnrc     @VKCOM/vk-sec
```